### DISCUSSION TITLE
Why are we using bluebird for creating functions that return promise instead of using native node js promise

### DISCUSSION THREAD

**Discussion by rogdevil** [worknenjoy/gitpay#978] [General]
Hey @alexanmtz I saw that we are using bluebird from creating functions that return promises, I was curious to know why. As I read on bluebird's github page they suggest to use native api instead of use bluebird's promises 
[here they mentioned it in a note](https://www.npmjs.com/package/bluebird#note)
please enlighten us ... thanks for your time : )

**Comment by alexanmtz** [comment]
Hey @rogdevil , this was an old implementation, and I guess we can migrate, and I will create an issue for making this refactoring in the whole application.

**Reply by rogdevil** [reply]
amazing let me know if u need any help I would gladly do that @alexanmtz

**Comment by alexanmtz** [comment]
Here the issue @rogdevil if you want to jump in
https://github.com/worknenjoy/gitpay/issues/1191
